### ISSUE TITLE
Log message - Windows 10 - PF 20181118

### ISSUE THREAD

**Issue by phweyland** [aferrero2707/PhotoFlow#178]
Browsing for opening file I see a bunch of this message:

`(photoflow.exe:1840): GLib-GIO-CRITICAL **: 12:31:07.109: g_content_type_is_a: assertion 'type != NULL' failed`

**Comment by aferrero2707** [comment]
This seems to be related to language settings: https://github.com/Alexpux/MINGW-packages/issues/3618

Does the folders contain file names with special characters? I still need to figure out why this is a problem under Windows, but not under Linux or MacOS...

**Comment by phweyland** [comment]
Yes, this is same log message as on Alexpux/MINGW-packages#3618. That happens while browsing file, which is done successfully except for folders with special character which do not appear. Files with special characters do appear.
However opening a file with special character fails. I've commented the issue #153 yesterday ...  
Let me know if I can do anything to help to solve this. :-)

**Comment by phweyland** [comment]
As a temporary work around is there a way to disable the console messages (the more I have files in the folder, the longer I've to wait :-) ) ?

**Comment by aferrero2707** [comment]
@phweyland Still working on the non-ASCII character support under Windows, that's something I really don;t grasp yet.
Unfortunately I do not know if any simple way to disable the console messages, sorry...

**Comment by ComputingFroggy** [comment]
running meld from git bash window I am still affected by this bug, any workaround  to fix it ?
